includes/class-wc-stripe-blocks-support.php

```diff
@@ -174,6 +183,7 @@ public function get_payment_method_data() {
 		return array_replace_recursive(
 			$this->get_gateway_javascript_params(),
 			$this->get_payment_request_javascript_params(),
+			$this->get_express_checkout_javascript_params(),
```

Comment:
@Mayisha Had some time to closely look at the empty box/403 forbidden problem I reported earlier, and it looks like it's from this line ^ that's not checking whether ECE is enabled or not before replacing JS (nonce) params

Reply:
Thanks @annemirasol for the findings 😃